src/main/java/CommandLineInterpreter.java

```diff
@@ -0,0 +1,80 @@
+package duke;
+import java.util.List;
+import duke.task.*;
```

Comment:
what you can do to prevent intellij from auto importing * is to change the settings in intellij by going to
File > settings > code style > java > imports > change the following:
Class count to use import *; 
Names count to use static import with *;
to some large enough number